use-cases/bed-allocation/src/main/java/org/acme/bedallocation/domain/Schedule.java

```diff
@@ -0,0 +1,133 @@
+package org.acme.bedallocation.domain;
+
+import java.util.List;
+
+import ai.timefold.solver.core.api.domain.solution.PlanningEntityCollectionProperty;
+import ai.timefold.solver.core.api.domain.solution.PlanningScore;
+import ai.timefold.solver.core.api.domain.solution.PlanningSolution;
+import ai.timefold.solver.core.api.domain.solution.ProblemFactCollectionProperty;
+import ai.timefold.solver.core.api.domain.valuerange.ValueRangeProvider;
+import ai.timefold.solver.core.api.score.buildin.hardmediumsoft.HardMediumSoftScore;
+import ai.timefold.solver.core.api.solver.SolverStatus;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@PlanningSolution
+public class Schedule {
+
+    private List<Department> departments;
+
+    @PlanningEntityCollectionProperty
+    private List<BedDesignation> bedDesignations;
+
+    @PlanningScore
+    private HardMediumSoftScore score;
+
+    private SolverStatus solverStatus;
+
+    // No-arg constructor required for Timefold
+    public Schedule() {
+    }
+
+    public Schedule(HardMediumSoftScore score, SolverStatus solverStatus) {
+        this.score = score;
+        this.solverStatus = solverStatus;
+    }
+
+    // ************************************************************************
+    // Getters and setters
+    // ************************************************************************
+
+    @JsonIgnore
+    @ProblemFactCollectionProperty
+    public List<String> getSpecialisms() {
+        return this.departments.stream().flatMap(d -> d.getSpecialismsToPriority().keySet().stream()).distinct().toList();
+    }
```

Comment:
Is this ever used anywhere? I don't see it used in the constraints and if it's not in the constraints, it doesn't need to exist.